Xamarin.PropertyEditing/ViewModels/PropertyViewModel.cs

```diff
@@ -411,6 +411,11 @@ public ICommand NavigateToValueSourceCommand
 
 		public virtual bool SupportsValueSourceNavigation => false;
 
+		public virtual ValueSource ValueSource
```

Comment:
This should have been left abstract, its value must be valid in all cases, an implementing subclass can not leave this as default.